### ISSUE TITLE
RUN AS ROOT

### ISSUE THREAD

**Issue by NetworkSpoiler** [Cyber-Dioxide/Wifi-Brute#1]
After i run the last command it says that i need to run the tool as root.

**Comment by Cyber-Dioxide** [comment]
It works on rooted android phones.

**Comment by sciborg** [comment]
My a12 is not rooted and I am getting same thing after last command

**Comment by Cyber-Dioxide** [comment]
> My a12 is not rooted and I am getting same thing after last command

Root your device for wireless attacks

**Comment by Cheeth05** [comment]
hello sir thank you for the tutorial but how to root the phone sir?

**Comment by Eftee-hacker** [comment]
I have rooted device , but it seems that run as root ..
After it what should i command

**Comment by Manto321** [comment]
yeah but... I used _fakeroot_ or *fakeroot* to run it... But the python3 files not working
